components/navbar/Navbar.tsx

```diff
@@ -41,7 +41,7 @@ const Navbar = () => {
       </div>
       {open && (
         <div
-          className={`fixed inset-y-0 right-0 z-50 w-[53%] bg-white dark:bg-gray-800 sm:hidden ${open ? "translate-x-0" : "translate-x-full"} transition-transform duration-300 ease-in-out`}
+          className={`fixed inset-y-0 right-0 z-50 w-[53%] bg-white transition-transform duration-300 ease-in-out dark:bg-gray-800 lg:hidden`}
```

Comment:
Do not use `gray` color. Use `secondary` instead. This is because theme is now configurable.